api/image_pipeline/pipeline_image_names.py

```diff
@@ -0,0 +1,235 @@
+from api import exceptions
+from api.image_pipeline import pipeline_image_util
+from typing import Dict, Tuple
+from api.image_pipeline.classes import Github, Distgit, Brew, CDN, Delivery
+
+VARIANT_BASE = "8Base-RHOSE"
```

Comment:
mmm, as of 4.12 this assumption will be broken... `ironic` images _should_ be in the 9Base variant (though TBH I'm not sure we've got that configured yet :grimacing:). the only sign in ocp-build-data is that it builds with a different `distgit.branch`... which needs to be mapped through `erratatool.yml` `brew_tag_product_version_mapping` for the product version and from there somehow to the variant (there's only one each I think). That might be difficult enough to defer it to a follow-on action item...

Reply:
Got it. Created an issue to keep track of it.